CONTRIBUTING.md

```diff
@@ -128,6 +128,86 @@ Two other things to keep in mind:
 - If you add code that should be tested, add tests.
 - If you change the public API, [update the documentation](#api-reference).
 
+### Debugging Environment
```

Comment:
Adding a little more explanation here would be good. We should talk about how this attaches two debuggers to the same process, and the rust debugger will break on rust breakpoints and the python debugger will break on python breakpoints.

Reply:
Yeah I agree. Thanks for all of the comments/suggestions!

Reply:
Just pushed a commit with a details/description section.